todo/src/index.ts

```diff
@@ -1,3 +1,4 @@
+/*
 import { Server, Request, ResponseToolkit } from "@hapi/hapi";
 const init = async () => {
```

Comment:
Why is this being commented out

Reply:
I decided to write a different way of handling the server and the routes and just forgot to remove the file when I committed.